100-level-AsyncAPI-Intro/step5.md

```diff
@@ -0,0 +1,20 @@
+In this intergalactic journey, you explored AsyncAPI, a powerful specification for crafting asynchronous APIs. The first module discusses what AsyncAPI is and why to use AsyncAPI, exploring real-world AsyncAPI use cases.
+
+Building on the basics, module two transitions into understanding Event-driven Architecture (EDA), showcasing how AsyncAPI fuels the design of both EDA and microservices.
+
+The third module explores the AsyncAPI specification. You learn about the structure of AsyncAPI and its different components, such as info, channels, components, and tags. You also learn to define operations, messages, and schemas, and the request/reply pattern.
+
+The fourth module focuses on hands-on experience with exercises for writing AsyncAPI code and documents from scratch. It also introduces AsyncAPI Studio for validating your documents.
```

Comment:
```suggestion
The fourth module focuses on hands-on experience with exercises for writing AsyncAPI documents from scratch and generating code from them. It also introduces AsyncAPI Studio for validating your documents.
```

Reply:
BTW, nothing about generating code is explained atm in this module. See https://github.com/asyncapi/learning-paths/pull/4#issuecomment-1952469968

Reply:
I would have to ask the author of module 4 for this, let me ping her.

Reply:
Yes, Sergio is correct that Module 4 lacks the "code generation" content. (which will be addressed in that PR)

However, this PR is for module 5 and Hrideysh is correct in noting that module 4 should cover code generation. IOW, nothing for Hridyesh to correct, it's Mahfuza's work that needs correction. 👍🏻